rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -628,6 +698,146 @@ CFontTexture::~CFontTexture()
 #endif
 }
 
+/***
+ *
+ * Add a fallback font
+ *
+ * @param fontfile VFS path for the font
+ */
+bool CFontTexture::AddFallbackFont(const std::string& fontfile)
+{
+#if defined(USE_FONTCONFIG) && !defined(HEADLESS)
+	if (!FtLibraryHandler::CanUseFontConfig())
+		return false;
+
+	FcFontSet *set = FtLibraryHandler::GetGameFontSet();
+
+	// Check if font already loaded
```

Comment:
The above change lets games pick which of two fonts under the same path (but different VFS mode) is loaded. Loading both of them still won't be possible due to this check. The check is doing important work though, and I don't expect the limitation to be a big problem, should be easy to work around if desired, so as long as it's mentioned in the Lua API doc it's IMO good enough.